src/route-handlers/start-workflow/start-workflow.ts

```diff
@@ -79,7 +132,11 @@ export async function startWorkflow(
       workflowId: resolvedWorkflowId,
     });
   } catch (e) {
-    logger.error<RouteHandlerErrorPayload>(
+    const isDuplicateWorkflowError =
+      e instanceof GRPCError && e.grpcStatusCode === status.ALREADY_EXISTS;
+
+    const logMethod = isDuplicateWorkflowError ? 'info' : 'error';
```

Comment:
Would be better to write a completely different log message for "already exists" than change only the log method